app/controllers/outfits_controller.rb

```diff
@@ -0,0 +1,25 @@
+class OutfitsController < ApplicationController
+  def index
+    @outfits = Outfit.all
+  end
+
+  def new
+    @outfit = Outfit.new
+  end
+
+  def create
+    @outfit = Outfit.new(outfit_params)
+
```

Comment:
we need to assign the user to the outfit:
`@user = User.find(params[:user_id])
@outfit.user = @user
`